clang/test/Driver/sycl-offload-new-driver.c

```diff
@@ -186,3 +186,10 @@
 // RUN:          -fno-sycl-libspirv -nocudalib --offload-new-driver %s 2>&1 \
 // RUN:   | FileCheck -check-prefix NVPTX_DEF_ARCH %s
 // NVPTX_DEF_ARCH: clang-offload-packager{{.*}} "--image=file={{.*}},triple=nvptx64-nvidia-cuda,arch=sm_50,kind=sycl"
+
+/// check for -sycl-embed-ir transmission to clang-linker-wrapper tool
+// RUN: %clangxx -fsycl -### -fsycl-targets=nvptx64-nvidia-cuda \
+// RUN:          -fno-sycl-libspirv -nocudalib --offload-new-driver \
+// RUN:          -fsycl-embed-ir %s 2>&1 \
```

Comment:
What happens if `-fsycl-embed-ir` is passed for devices other than NVidia or AMD?

Reply:
According to our implementation, it will be ignored. I can add a test for it.

Reply:
Is that acceptable? Would a warning or error be appropriate here?